### ISSUE TITLE
Stop using a model that may potentially not exist

### ISSUE THREAD

**Issue by skipperguy12** [NJayDevelopment/mongoid_forums#17]
https://github.com/NJayDevelopment/mongoid_forums/blob/master/app/controllers/mongoid_forums/admin/groups_controller.rb#L57

This is occasionally used, especially in admin where we deal directly with User. It should be `MongoidForums.user_class.find(...)`

**Comment by AustinLMayes** [comment]
@skipperguy12 what if the user doesn't have an id or the app is using a different method overall to find users, you might want to add a method to the user class that acts as an identifier.

**Comment by skipperguy12** [comment]
@AustinLMayes good point, I will do what forem has done when I get a chance. They have an auto complete_field.
